src/caret_analyze/record/records_service/response_time.py

```diff
@@ -450,33 +450,46 @@ def __init__(
         self._histogram = ResponseHistogram(self._records, self._timeseries)
 
     def to_all_records(self) -> RecordsInterface:
-        return self._response_map_all.to_all_records()
+        """
+        Calculate the all records data for response time.
```

Comment:
```suggestion
        Calculate the data of all records for response time.
```
or
```suggestion
        Calculate all the records' data for response time.
```

Reply:
Fixed it in [a103900](https://github.com/tier4/CARET_analyze/pull/355/commits/a1039004f34c90c7c7f17675a466049aaf7220d5).